mkdocs/docs/protocol/onchain-credential-status.md

```diff
@@ -0,0 +1,190 @@
+# OnChain Credential Status
+
+On-chain identity verification requires the ability to check whether a claim has been revoked using a smart contract. To accomplish this, the claim must include information about the smart contract, such as its address, blockchain, and network. This information can be stored in the `CredentialStatus` structure.
+
+The `CredentialStatus` structure should contain the following fields:
+
+```golang
+type CredentialStatus struct {
+	ID                 string               `json:"id"`
+	Type               CredentialStatusType `json:"type"`
+	RevocationNonce    uint64               `json:"revocationNonce"`
+}
+```
+
+The `ID` field is a composite field that contains encoded information in the following format:
+
+`did:[did-method]:[blockchain]:[network]:[id]/credentialStatus?(revocationNonce=value)&(contractAddress={chainID}:{contractAddress})`
+
+`type` - credential type
```

Comment:
`type` - credential type, in this case should be `Iden3OnchainSparseMerkleTreeProof2023`